src/components/Table/Table/TableConsts.ts

```diff
@@ -5,4 +5,9 @@ export enum RowSizes {
   LARGE = "large"
 }
 
-export type RowSize = typeof RowSizes | number;
+export enum RowHeights {
+  medium = 40,
+  large = 48
+}
```

Comment:
I know we came up with this quickly, we'll need to think of some nicer solution other than this 😆

Reply:
Adding to our tech debts 🥲 
<img src="https://media2.giphy.com/media/xT9IgqYHYClWb2eZnG/giphy.gif"/>

Reply:
Yeah agree! Let me think about something real quick